css/main.css

```diff
@@ -51,6 +50,101 @@ header {
   color: white;
 }
 
+header {
+  display: flex;
+}
+
+.navbar {
+  justify-content: center;
+  align-items: center;
+  background-color: #000;
+  color: #fff;
+  border-bottom: 1px solid rgba(255,255,255,0.2);
+  width: 100%;
+  height: 80px;
+  line-height: 80px;
+  font-size: 18px;
+  padding: 0;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 10;
+}
+
+.navbar a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.navbar a.navbar-brand {
+  line-height: inherit;
+  font-size: 22px;
+  text-transform: uppercase;
+  font-weight: 400;
+  letter-spacing: 2px;
+}
+
+.navbar h1 {
+  padding-left: 20px;
+}
+
+.navbar ul {
+  width: 100%;
+  list-style: none;
+  line-height: inherit;
+  padding: 0;
+  background-color: rgba(0,0,0,0.9);
+  height: 60vh;
+  max-height: 0;
+  overflow: hidden;
+  transition: all ease-in-out 0.3s;
+}
+
+.navbar ul li {
+  display: inline-block;
+}
+
+.navbar ul li a {
+  display: block;
+  text-align: center;
+  min-width: 120px;
+  width: 100%;
+  padding: 0 30px;
+}
+.navbar ul li a:hover {
+  background-color: rgba(255,255,255,0.15);
+}
+
+.navbar .navbar-toggler {
+  display: none;
+}
+
+.navbar ul.open {
+  max-height: 100vh;
+}
+
+.navbar ul li {
+  width: 100%;
+}
+
+.navbar .navbar-toggler {
+  display: block;
+  position: absolute;
+  top: 20px;
+  right: 20px;
+  background-color: transparent;
+  color: #fff;
+  border: 3px solid #fff;
+  outline: none;
+  padding: 8px;
+  cursor: pointer;
+}
+
+.navbar-toggler .fa-times {
+  display: none;
+}
+
+
```

Comment:
- [ ] Again, please remove the blank space over there. Also, look at the other comment above about the same issue.